### ISSUE TITLE
many Lua errors

### ISSUE THREAD

**Issue by TrinooLive** [Questie/Questie#5273]
3257x ...ddOns/Questie/Modules/FramePool/QuestieFramePool.lua:216: Wrong object type for function
[string "=[C]"]: in function `CreateFrame'
[string "@Questie/Modules/FramePool/QuestieFramePool.lua"]:216: in function `CreateLine'
[string "@Questie/Modules/FramePool/QuestieFramePool.lua"]:186: in function `CreateWaypoints'
[string "@Questie/Modules/Map/QuestieMap.lua"]:961: in function `DrawWaypoints'
[string "@Questie/Modules/Map/QuestieMap.lua"]:400: in function `ShowNPC'
[string "@Questie/Modules/QuestieMenu.lua"]:116: in function <Questie/Modules/QuestieMenu.lua:109>

Locals:
(*temporary) = "Button"
(*temporary) = "questieLineFrame6"
(*temporary) = <table> {
 1 = 62.298289
 2 = 17.286064
}

## Screenshots
![2023-12-12_000588](https://github.com/Questie/Questie/assets/80533377/ff95a428-db3d-4df7-b613-cb4b8fc3b604)
![2023-12-12_000589](https://github.com/Questie/Questie/assets/80533377/e5509780-001c-4dbc-9cbe-050acd1aa544)

## Questie & WoW version
Questie v9.2.0
WoW Classic SOD

**Comment by Duraznos** [comment]
@TrinooLive i got it to work by going to `Questie\Modules\Map\QuestieMap.lua` and deleting line 398 that says `manualIcons[zone] = { waypoints[1][1][1], waypoints[1][1][2] }`

@Muehe it looks like that line was added in error because you assign to `manualIcons[zone]` in the line right above it

```lua
                if not manualIcons[zone] then
                    manualIcons[zone] = QuestieMap:DrawManualIcon(data, zone, waypoints[1][1][1], waypoints[1][1][2])
                    manualIcons[zone] = { waypoints[1][1][1], waypoints[1][1][2] } -- you're throwing away the results of the prior function call
                end
```

**Comment by Muehe** [comment]
@Duraznos Just pushed the fix (5ac4fadaa0bb751f3ee12135d343c033af48cf05), but thanks anyway for pointing. :+1: 

Release is up on GitHub and should be up on Curseforge shortly.
